tests/test_analyzer.py

```diff
@@ -57,19 +57,14 @@ def mock_get_state_variable(self, name):
     def mock_get_list_of_models(self):
         return ['model1']
 
-    @patch.multiple(
```

Comment:
Are there any real changes to this file? Or just formatting?

Reply:
analysis_models was removed